example-conf/mesosphere-dcos/hdfs-site.xml

```diff
@@ -69,7 +69,7 @@
 
   <property>
     <name>ha.zookeeper.quorum</name>
-    <value>master0.mesos:2181,master1.mesos:2181,master2.mesos:2181</value>
```

Comment:
Shouldn't we keep 3 masters as the default since production level requires a quorum of 3?